src/modules/refresh.js

```diff
@@ -30,6 +28,7 @@ const refreshList = (_class, _container) => {
         listIconDots.classList.add('hide');
         removeBtn.classList.remove('hide');
         todoList.style.backgroundColor = '#fffdcc';
+        listIconTrash.style.backgroundColor = '#F50000';
```

Comment:
- [ ] hello and nice job on adding the requested change by the previous CR, but the complete status doesn't change after selecting it as the `if (e.completed === true) {
      isChecked = 'checked';
      strikeThrough = 'strike-through';
    }` it stays false all the time


![Capture web_9-7-2022_171827_gitlab com](https://user-images.githubusercontent.com/61562626/178114109-f4fa65f2-126d-402c-982d-70b46c657beb.jpeg)

Reply:
I do not understand what you are saying